ui/app/src/assets/uiConfigDefaults.ts

```diff
@@ -494,6 +494,23 @@ const uiConfigDefaults = {
               }
             ]
           }
+        },
+        {
+          id: 'craftercms.components.ToolsPanelPageButton',
+          uiKey: count++,
+          roles: ['admin', 'developer'],
+          configuration: {
+            title: 'Settings',
```

Comment:
Use message descriptor.
In fact, please change all raw texts to message descriptors.